src/components/Button/Button.scss

```diff
@@ -149,6 +149,13 @@ $spinner-size: rem(20px);
     margin-right: 0;
   }
 
+  &.destructive {
+    color: color('red');
+    &:hover {
+      color: color('red', 'dark');
```

Comment:
@ry5n do you have any thoughts on creating a `destructive` style for a `plain` Buttons. Any potential downsides to this? Omitting it feels like an oversight to me. Also, what do you think about the hover style in this implementation?